### PR TITLE
Updating to not use the directory

### PR DESCRIPTION
**Pull request by glennblock**

### CODE REVIEW

File: content/src/policies/__id/delete.rego [hunk 1]
```diff
@@ -1,21 +1,37 @@
 package peoplefinder.DELETE.api.users.__id
 
+# Policy for deleting an employee in PeopleFinder
+#
+# Note: This policy is for illustration. The Delete button visibility is used by PeopleFinder, but the button does not call the delete API.
```
Comment (ogazitt) [inline content/src/policies/__id/delete.rego]: same comment as in the other PR

File: content/src/policies/__id/post.rego [hunk 2]
```diff
@@ -1,19 +1,33 @@
 package peoplefinder.POST.api.users.__id
 
+# Policy for updating an employee's deparment and title in PeopleFinder
+#
+# The user is Allowed if:
+# - User is an admin
```
Comment (ogazitt) [inline content/src/policies/__id/post.rego:6]: maybe you could insert an "or" here, to make it clear that either or?

or, `# The user is Allowed if either is true:`

File: content/src/policies/post.rego [hunk 3]
```diff
@@ -1,19 +1,35 @@
 package peoplefinder.POST.api.users
 
+# Policy for creating an employee in PeopleFinder
+#
+# Note: This policy is for illustration and is not implemented within the PeopleFinder application.
```
Comment (ogazitt) [inline content/src/policies/post.rego]: ditto here

**Review by ogazitt — CHANGES_REQUESTED** [review]

File: content/src/policies/__id/post.rego (hunk 2 above)
Comment (ogazitt) [reply content/src/policies/__id/post.rego:6]: @glennblock

Comment (glennblock) [reply content/src/policies/__id/post.rego:6]: Done in both templates. Example of the format below.

```
# The user is Allowed if:
# - User is an admin
# - OR User is the manager or in the management chain of the employee```

**Review by ogazitt — APPROVED** [review]

**Pull request merged**